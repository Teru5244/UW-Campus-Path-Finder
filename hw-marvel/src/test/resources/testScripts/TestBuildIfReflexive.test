# Test if the graph is built reflexively between nodes

LoadGraph ref simpleGraph.csv

ListNodes ref

ListChildren ref n1
ListChildren ref n2
ListChildren ref n3