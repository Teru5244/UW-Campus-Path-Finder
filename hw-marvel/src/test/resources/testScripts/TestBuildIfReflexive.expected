# Test if the graph is built reflexively between nodes

loaded graph ref

ref contains: n1 n2 n3

the children of n1 in ref are: n2(e1) n3(e3)
the children of n2 in ref are: n1(e1) n3(e2)
the children of n3 in ref are: n1(e3) n2(e2)